### PR TITLE
update migration and seeder

### PR DESCRIPTION
**Pull request by hieudotrung**

### CODE REVIEW

File: app/Exam.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Exam extends Model
+{
+    protected $fillable = [
+        'name', 'quanlity', 'time', 'user_id', 'topic_id'
+    ];
+    public function topic(){
```
Comment (phamkykhoi) [inline app/Exam.php]: public function topic(){
=>
public function topic()
{

File: app/Exam.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    	return $this->belongsTo('App\Topic');
+    }
+    public function user_exam(){
```
Comment (phamkykhoi) [inline app/Exam.php]: public function user_exam(){
=>
public function user_exam()
{

File: app/Topic.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+use Illuminate\Database\Eloquent\Model;
+
+class Topic extends Model
+{
+    protected $fillable = [
+    	'name_topic'
+    ];	
+    
+    public function exams(){
```
Comment (phamkykhoi) [inline app/Topic.php]: public function exams(){
=>
public function exams()
{

File: app/User.php [hunk 4]
```diff
@@ -26,4 +26,7 @@ class User extends Authenticatable
     protected $hidden = [
         'password', 'remember_token',
     ];
+    public function User_exams(){
```
Comment (phamkykhoi) [inline app/User.php]: public function User_exams(){

=>

public function User_exams()
{

File: database/factories/ModelFactory.php [hunk 5]
```diff
@@ -15,9 +15,47 @@
     static $password;
 
     return [
-        'name' => $faker->name,
-        'email' => $faker->unique()->safeEmail,
-        'password' => $password ?: $password = bcrypt('secret'),
-        'remember_token' => str_random(10),
+		'username'       => $faker->name,
```
Comment (phamkykhoi) [inline database/factories/ModelFactory.php]: Indent

File: database/factories/ModelFactory.php [hunk 6]
```diff
@@ -15,9 +15,47 @@
...
+		'email'          => $faker->unique()->safeEmail,
```
Comment (phamkykhoi) [inline database/factories/ModelFactory.php]: Indent

File: database/factories/ModelFactory.php [hunk 7]
```diff
@@ -15,9 +15,47 @@
...
+		'password'       => $password ?: $password = bcrypt('secret'),
```
Comment (phamkykhoi) [inline database/factories/ModelFactory.php]: Indent

File: database/factories/ModelFactory.php [hunk 8]
```diff
@@ -15,9 +15,47 @@
...
+		'is_admin'       => $faker->randomElement([0,1]),
```
Comment (phamkykhoi) [inline database/factories/ModelFactory.php]: Indent

File: database/factories/ModelFactory.php [hunk 9]
```diff
@@ -15,9 +15,47 @@
...
+		'remember_token' => str_random(10),
```
Comment (phamkykhoi) [inline database/factories/ModelFactory.php]: Indent

File: database/factories/ModelFactory.php [hunk 10]
```diff
@@ -15,9 +15,47 @@
...
+// end vi du
+
+$factory->define(App\Exam::class, function (Faker\Generator $faker) {
+    return [
+		'name'     => $faker->name,
+		'quantity' => $faker->randomElement([10,20,30,40]),
+		'time' 	   => $faker->randomElement([45,10,15]),
+		'user_id' => 1,
+		'topic_id' => 1
+ 		// // 'user_id'  => function () {
```
Comment (dung13890) [inline database/factories/ModelFactory.php]: Xoa comment

File: database/factories/ModelFactory.php [hunk 11]
```diff
@@ -15,9 +15,47 @@
...
+		// // 	return factory(App\Users::class)->create()->id;
```
Comment (phamkykhoi) [inline database/factories/ModelFactory.php]: Remove comment

File: database/migrations/2016_11_23_024227_create_exams_table.php [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+     */
+    public function up()
+    {
+        Schema::create('Exams', function (Blueprint $table) {
+            $table ->increments('id');
+            $table ->string('name')->unique();
+            $table ->tinyInteger('quantity');
+            $table ->integer('time');
+            $table ->integer('user_id')->default(0);
+            // $table->foreign('user_id')->references('id')->on('users')->onDelete('cascade');
```
Comment (phamkykhoi) [inline database/migrations/2016_11_23_024227_create_exams_table.php]: Remove comment

File: database/migrations/2016_11_23_024227_create_exams_table.php [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+            $table ->integer('topic_id')->default(0);
+            // $table->foreign('topic_id')->references('id')->on('topics')->onDelete('cascade');
```
Comment (phamkykhoi) [inline database/migrations/2016_11_23_024227_create_exams_table.php]: Remove comment

File: database/seeds/ExamsSeeder.php [hunk 14]
```diff
@@ -0,0 +1,29 @@
...
+        $topic_id = App\Topic::get();
+        $user_id  = App\User::get();
+        factory(App\Exam::class, 50)->create()->each(function ($u) use ($topic_id, $user_id) {
+            $u->update([
+                    'topic_id' => $topic_id->random(1)->id,
+                    'user_id'  => $user_id->random(1)->id
+             ]);
+        });
+
+        //dd($topic_id);
```
Comment (phamkykhoi) [inline database/seeds/ExamsSeeder.php]: Remove

File: database/seeds/ExamsSeeder.php [hunk 15]
```diff
@@ -0,0 +1,29 @@
...
+        	
+        // });
```
Comment (phamkykhoi) [inline database/seeds/ExamsSeeder.php]: Remove

File: database/seeds/UsersTableSeeder.php [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        factory(User::class, 50)->create()->each(function($u) {
+        	if ($u->id === 1) {
+        		$u->update([
+        			'username' => 'admin',
```
Comment (phamkykhoi) [inline database/seeds/UsersTableSeeder.php]: Indent

File: database/seeds/UsersTableSeeder.php [hunk 17]
```diff
@@ -0,0 +1,25 @@
...
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        factory(User::class, 50)->create()->each(function($u) {
+        	if ($u->id === 1) {
```
Comment (dung13890) [inline database/seeds/UsersTableSeeder.php]: Căn chỉnh lại indentation

**Pull request merged**